podvm/README.md

```diff
@@ -0,0 +1,128 @@
+# Introduction
+
+This directory contains the sources to build the podvm image (qcow2 file) for various Linux distributions and cloud providers. So use
+the instructions on the next sections if you need to build your own image with changes to meet your requirements. Otherwise you can
+find [here](../docs/consuming-prebuilt-podvm-images.md) information on how to consume pre-built images.
+
+# How to build on locally
+
+In order to build locally you must [packer](https://www.packer.io/) and QEMU installed in your host. Apart from those, it requires the source trees and software mentioned on the [developer's guide](docs/DEVELOPMENT.md).
+
+Run the following commands to build locally:
+
+```
+$ export CLOUD_PROVIDER=[aws|azure|ibmcloud|libvirt]
+$ make binaries
+$ make image
+```
+
+# How to build within container
+
+This directory contains dockerfiles to build the podvm image entirely within container so that it only requires docker or podman installed on the host.
+
+In general it is needed to follow these steps:
+
+1. Build a builder container image for a given Linux distribution
```

Comment:
Ideally we can have most builders publicly available, but i think we didn't push them

Reply:
Yeah that was the intent. Having podvm_builder images published for releases (including any pre-releases)

Reply:
Agreed. Actually I created this README aiming at developers that need to build podvm from latest code. And the [docs/consuming-prebuilt-podvm-images.md](https://github.com/confidential-containers/cloud-api-adaptor/pull/568/files/d5dab40a507fcd557cfa9747547ee6014a77dc78#diff-4fc2fb53a97f8dc41a360dc7a32b8dda5ff90f13819ce81e06f6c4e373e9e956) target consumers of the project.